src/client/static/index.tsx

```diff
@@ -43,7 +64,27 @@ if (window.Cypress) {
     if (country) {
       cmp.init({ country });
     }
+
+    initGoogleAnalyticsWhenConsented();
+    ophanInit();
   })();
 }
 
-initGoogleAnalyticsWhenConsented();
+loadableReady(() => {
+  const config = setupConfig();
+
+  sourceAccessibilityInit();
+
+  const params = new URLSearchParams(window.location.search);
+
+  if (params.has('useIslets')) {
+    console.log('init islets');
+    import('./islet').then(({ init }) => {
+      init();
+    });
+  } else {
+    import('./hydration').then(({ hydrateApp }) => {
+      hydrateApp(config);
+    });
+  }
```

Comment:
I wonder if it would be best to load `./islet` and `./hydration` as part of the main bundle instead of loading them asynchronously using `import()`. I'm just concious that this would mean at least two bundles would need to be loaded and ran before the islet bundles are requested.

Reply:
That's not what's happening. It only loads the `./islet` or `./hydration` depending on the environment it's running in, i.e whether we should use islets, or whether we should hydrate the whole bundle (as what's happening now). Once Islets are live we won't have the need for the `./hydration` file, so can move the `./islet` into the `main` bundle then.